jest/jest.setup.js

```diff
@@ -52,27 +52,37 @@ jest.mock('libs/environment', () => ({
 }))
 
 const originalWarn = console.warn.bind(console.warn)
-console.warn = function (message) {
+console.warn = function (...messages) {
   const messagesToIgnore = [
     /* React Native is not a real css environment. One of the things it doesn't support is nesting and actually selectors in general
     Source : https://github.com/styled-components/styled-components/issues/989#issuecomment-314946541 */
     'Node of type rule not supported as an inline style',
   ]
-  for (messageToIgnore of messagesToIgnore) {
-    if (message.includes(messagesToIgnore)) {
-      return
+
+  for (let message of messages) {
+    if (typeof message !== 'string') {
+      message = JSON.stringify(message)
+    }
+    for (messageToIgnore of messagesToIgnore) {
+      if (!message.includes(messagesToIgnore)) {
+        originalWarn(message)
+      }
     }
```

Comment:
Une des bone pratiques cités dans Clean Code est d'éviter les conditionnelles négatives par soucis de compréhension. Qu'en penses-tu ?
```suggestion
      if (message.includes(messagesToIgnore)) {
        return
      }
      originalWarn(message)
    }
```

Reply:
je sais que les doubles négations sont à proscrire parce qu'elles prennent du temps à être interprétées, mais les conditionnelles négatives je ne vois pas pourquoi.

je vais utiliser:
```ts
   if (message.includes(messagesToIgnore)) {
        break
      }
      originalWarn(message)
    }
```

Reply:
C'est ok pour moi. 
Et pour l'explication : https://csharpdeveloper.wordpress.com/2016/01/20/roberts-rules-of-coders-8-avoid-negative-conditionals-and-method-names/